src/main/java/pl/tomaszdziurko/jvm_bloggers/SpringApplicationContext.java

```diff
@@ -0,0 +1,21 @@
+package pl.tomaszdziurko.jvm_bloggers;
+
+import org.springframework.beans.BeansException;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SpringApplicationContext implements ApplicationContextAware {
+
+    private static ApplicationContext context;
+
+    @Override
+    public void setApplicationContext(ApplicationContext context) throws BeansException {
+        SpringApplicationContext.context = context;
+    }
+
+    public static <T> T getBean(Class<T> clazz) {
```

Comment:
I think it is to get use this bean in a `BlogPostEntityListener`, which is created and executed not in the Spring managed scope.